repository-data/local/src/main/resources/hcm-config/main.yaml

```diff
@@ -2,7 +2,6 @@
 definitions:
   config:
     /hippo:configuration/hippo:modules/autoexport/hippo:moduleconfig:
-      autoexport:enabled: false
```

Comment:
this file change/file can be excluded from the commit.

Reply:
Thanks @ngta1 will get this removed.

Reply:
@ngta1 should this go in a `.gitignore`?

Reply:
We usually keep the file in as it sets autoexport to false by default.  Occasionally when we add a new document type/test data, we need autoexport to be switched on to spit out the YAML bootstrap files needed.